src/middlewared/middlewared/plugins/filesystem.py

```diff
@@ -552,6 +558,9 @@ def statfs(self, path):
             'free_bytes': st.f_bfree * st.f_frsize,
             'avail_bytes': st.f_bavail * st.f_frsize,
         }
+        for k in ['total_blocks', 'free_blocks', 'avail_blocks', 'total_bytes', 'free_bytes', 'avail_bytes']:
```

Comment:
Can we add a comment about why we're doing this? 53 bit integer limitation for JSON isn't very well-known.

Reply:
@yocalebo I was thinking about doing that too, but I am not sure where. The same comment everywhere we do that? No. Can we just add this to some doc section?

Reply:
Meh, I think the git commit explanation is good enough for any developer that comes along afterwards and wants to know why these fields exist.

Reply:
git commit message will be lost with a refactoring commit, I'll add this to the docs